gql/builder.go

```diff
@@ -377,31 +389,46 @@ func ResolveListField(name string, parent string) graphql.FieldResolveFn {
 			return nil, err
 		}
 
+		sortParams, err := parseSortParameters(p.Args[sortArgumentName])
+		if err != nil {
+			return nil, err
+		}
+
 		resolve := ResolveByField(name, parent)
 
 		resolvedValue, err := resolve(p)
 		if err != nil {
 			return nil, err
 		}
-
-		if filter == nil {
```

Comment:
This could stay as a `if filter == nil && sortParams == nil` check if we wanted to avoid the copy into `values` for the (common) case of not having either a sort or a filter.

Reply:
good idea that is a simple optimization